insights/parsers/cpuinfo.py

```diff
@@ -243,7 +243,16 @@ def core_total(self):
         str: Returns the total number of cores for the server if available, else None.
         """
```

Comment:
@iphands can you please add the deprecation feature to this method?  See this [document](https://insights-core.readthedocs.io/en/latest/api.html?#feature-deprecation).  Also please provide a warning in the docstring to notify any users that `insights.parsers.lscpu.LsCPU.info['Cores per socket']` should be used instead, here is an [example](https://github.com/RedHatInsights/insights-core/blob/827faf50b5b34d78a2fd21167d69b9ea2aff3abc/insights/parsers/nfs_exports.py#L142).

Reply:
Bob feel free to add it man.

Reply:
@iphands before I push those changes, can you fix your original commit by adding the `--signoff`?  See the [DCO results](https://github.com/RedHatInsights/insights-core/pull/2563/checks?check_run_id=618266058) for the necessary commands.

Reply:
For sure